src/tests/fableconfig.json

```diff
@@ -10,9 +10,8 @@
     "plugins": ["../plugins/nunit/Fable.Plugins.NUnit.dll"],
     "targets": {
         "netcore": {
-            "projFile": "project.fsx",
+            "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
-            "module": "commonjs",
```

Comment:
Why is this deleted? The default module system now is `es2015`. It won't be possible to run the tests on Mocha if `commonjs` is not used.